blueprint/reviews/review-likes.mson

```diff
@@ -0,0 +1,15 @@
+
+## reviewLikes (object)
++ id: 1 (required)
++ type: reviewLikes (required)
++ links (object)
```

Comment:
Kitsu will always return `id`, `type` and `links: self` for all models. All other properties are optional and may not be shown depending on the request queries.